srcpkgs/gst-libav/template

```diff
@@ -1,9 +1,9 @@
 # Template file for 'gst-libav'
 pkgname=gst-libav
-version=1.16.2
+version=1.18.0
 revision=1
-wrksrc="${pkgname}-${version}"
 build_style=meson
+configure_args="-Ddoc=disabled"
```

Comment:
Can't they detect that `hotdoc` isn't available?

Reply:
They do detect that `hotdoc` isn't available, which results in build error. `doc` is enabled by default in all of them, so if it `hotdoc` isn't found, build fails.

Reply:
Huh. Since they recommend against building docs, I'd expect their builds to not fail without the docs program. Oh well.